vadl/main/vadl/lcb/passes/relocation/GenerateLinkerComponentsPass.java

```diff
@@ -336,8 +338,12 @@ private static void genAbs(Format.Field imm,
     variantKinds.add(variantKind);
     linkModifierToVariantKind.add(Pair.of(modifier, variantKind));
 
+    // THIS IS A HACK
+    // We need to implement it for all fields but only is supported at the moment.
+    var firstField = imm.fieldRefs().getFirst();
+
     var updateFieldFunction =
-        BitMaskFunctionGenerator.generateUpdateFunction(format, imm);
+        BitMaskFunctionGenerator.generateUpdateFunction(format, firstField);
```

Comment:
fyi @benjaminkasper99 

Here we have to generate something like in the encoder wrapper